tests/pfcwd_masic_test.py

```diff
@@ -0,0 +1,167 @@
+Ethernet-BP256:14       N/A                        0/1       3252/68       6306/55               78/7            7846/99
+Ethernet-BP256:15       N/A                        1/0        968/15       8387/98               15/2            6495/35
+ Ethernet-BP260:0       N/A                        1/0       2030/40       6262/29              922/2            4880/84
+ Ethernet-BP260:1       N/A                        0/1        935/94       8236/44              139/0             3945/6
+ Ethernet-BP260:2       N/A                        0/0       3989/29       4700/54              250/3            5431/27
+ Ethernet-BP260:3       N/A                        0/0       8774/59       5977/88              853/6             123/22
+ Ethernet-BP260:4       N/A                        1/1       3289/58       8762/40              509/9            5097/75
+ Ethernet-BP260:5       N/A                        1/0       3083/84       1933/90              818/8             590/24
+ Ethernet-BP260:6       N/A                        1/0       1460/71        5827/5              775/5            9836/77
+ Ethernet-BP260:7       N/A                        0/1       7692/28        3935/8              299/3            2368/75
+ Ethernet-BP260:8       N/A                        1/1       7370/87       6133/16              581/1            7944/62
+ Ethernet-BP260:9       N/A                        1/1       5719/87       1561/11              235/1            4054/49
+Ethernet-BP260:10       N/A                        1/0        6526/9       5217/26              260/4            9338/69
+Ethernet-BP260:11       N/A                        1/1       5615/81       9506/12              185/5            9902/28
+Ethernet-BP260:12       N/A                        1/1         76/28       4344/45              532/7            8590/69
+Ethernet-BP260:13       N/A                        0/1       3839/80       8143/47              601/7             575/97
+Ethernet-BP260:14       N/A                        1/1       3787/72        9865/4              493/6             537/95
+Ethernet-BP260:15       N/A                        0/1       6803/57        804/71              104/3            8343/72
+"""
+
+show_pfc_config_all = """\
+Changed polling interval to 199 ms on asic0
+BIG_RED_SWITCH status is enable on asic0
+Changed polling interval to 199 ms on asic1
+BIG_RED_SWITCH status is enable on asic1
+          PORT    ACTION    DETECTION TIME    RESTORATION TIME
+--------------  --------  ----------------  ------------------
+     Ethernet0      drop               200                 200
+     Ethernet4      drop               200                 200
+  Ethernet-BP0      drop               200                 200
+  Ethernet-BP4      drop               200                 200
+Ethernet-BP256      drop               200                 200
+Ethernet-BP260      drop               200                 200
+"""
+
+
+class TestMultiAsicPfcwdShow(object):
```

Comment:
Can you include cases to cover all the options for 'show stats' and 'show config'. display for a namespace, display when a set of  ports/queues are included?

Reply:
The "display" option will not work. Requires a bit of explanation but it has to do with 'multi-asic-click-options' which populates the parameters before the environment variables are set un 'setup()' method. Due to this the multi asic namespace options are not available when the pytest framework loads all the .py files. Tried several options including 'reload', but it didn't work. We are dicussing internally what other options are available including changing implementation of 'multi-asic-click-option'.

Reply:
Would these changes come in a following PR?

Reply:
Yes.

Reply:
#1093